hw/ip/aes/data/aes.prj.hjson

```diff
@@ -7,5 +7,5 @@
     version:            "0.5",
     life_stage:         "L1",
     design_stage:       "D2",
-    verification_stage: "V0",
+    verification_stage: "V1",
```

Comment:
Would it be okay to hold off on merging this PR, since there are V1 items that are still "in progress" and "not started"?

Reply:
Actually, the this is the idea of this PR. We need to trigger the review. We would like to know what is missing to push this forward.

Reply:
Sorry @sriyerg I did not realize that the V1 review is separate process from the testplan review. We would like to move on, if possible let's do this in one shot.